### DISCUSSION TITLE
LocalHost Webfilter,

### DISCUSSION THREAD

**Discussion by AzlanCoding** [abhinavsingh/proxy.py#1315] [Q&A] [answered]
Hello, I would like to use proxy.py to filter websites and it's content (such as vulgarities). Is this possible with python and on localhost?

**Comment by abhinavsingh** [comment, marked as answer]
Hi @AzlanCoding , yes there are existing plugin example which demonstrate how to filter content.   Please take those skeleton example to introduce your own custom filtering.  Example, https://github.com/abhinavsingh/proxy.py#filterbyupstreamhostplugin is one such plugin.

**Comment by AzlanCoding** [comment]
Is there a way to redirect the user to another website instead of returning http 418?

**Reply by abhinavsingh** [reply]
Please see https://github.com/abhinavsingh/proxy.py#redirecttocustomserverplugin
